pkgs/by-name/lo/local-content-share/package.nix

```
@@ -0,0 +1,30 @@
+{
+  lib,
+  buildGoModule,
+  fetchFromGitHub,
+}:
+
+buildGoModule rec {
```

```suggestion
buildGoModule (finalAttrs: {
```

The `finalAttrs` pattern is usually preferred in places of rec because it allows for [overriding](https://nixos.org/manual/nixpkgs/unstable/#mkderivation-recursive-attributes).